libs/priors/priors/priors.py

```diff
@@ -15,7 +15,8 @@
     Uniform,
 )
 from bilby.gw.prior import UniformComovingVolume, UniformSourceFrame
-from priors.utils import (
+
+from aframe.priors.utils import (
```

Comment:
Does this change work? There's no longer an intermediate `aframe` directory in any of the libs.